source/agora/consensus/protocol/Nominator.d

```diff
@@ -239,7 +239,7 @@ extern(D):
 
         // Find the node id of this validator and create an SCPObject
         const this_utxo = this.enroll_man.getEnrollmentKey();
-        NodeID node_id = ledger.getValidators(Height(1))
+        NodeID node_id = ledger.getValidators(ledger.height + 1)
```

Comment:
```suggestion
        NodeID node_id = this.ledger.getValidators(this.ledger.height + 1)
```

Reply:
As `ledger` is also a parameter I think it is ok.